server/api/api/endpoints/functions.py

```diff
@@ -580,6 +592,40 @@ def _handle_job_deploy_status(
         )
 
     if (
+        logs
+        and normalized_pod_function_state == mlrun.common.schemas.FunctionState.pending
+        and server.api.utils.helpers.validate_client_version(
+            client_version, "1.8.0-rc1"
+        )
+    ):
+        response_headers["deploy_status_text_kind"] = (
+            mlrun.common.constants.DeployStatusTextKind.events
+        )
+        build_pod_events = server.api.utils.singletons.k8s.get_k8s_helper(
+            silent=False
+        ).list_object_events(object_name=pod)
+        logger.debug(
+            "Resolved build pod events",
+            function_name=name,
+            pod=pod,
+            events_len=len(build_pod_events),
+        )
+        resp = ""
+        for event in build_pod_events:
+            event: kubernetes.client.CoreV1Event
+            if event.type == "Normal":
+                # Do not spam with normal events
+                continue
```

Comment:
Is this the only OK event type?
So we want just print events that are not interesting.

Reply:
according to https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/
> type is the type of this event (Normal, Warning), new types could be added in the future. It is machine-readable. This field cannot be empty for new Events.